ddtrace/contrib/datadog_lambda/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+The datadog_lambda integration traces timeout spans in an
+AWS Lambda function instrumented with the
+`Datadog Lambda Python <https://github.com/DataDog/datadog-lambda-python>`_ package.
+
+Enabling
+~~~~~~~~
+
+The datadog_lambda integration is enabled automatically for AWS Lambda
+functions which meet one of the following conditions::
+
+    1. The environment variable ``DD_LAMBDA_HANDLER`` is set.
+
+    When this variable is set, it's asummed that the function was automatically
+    instrumented. Therefore, a Datadog Lambda Layer (``datadog-lambda-py``), and
+    a Datadog Extension Layer (``datadog-lambda-extension``) are set.
+
+    2. The AWS Lambda function was instrumented manually by following
+    the recommended `installation <https://docs.datadoghq.com/serverless/guide/handler_wrapper/>`_ by Datadog.
+
+
+The patch aims to get the arguments provided to the ``datadog_lambda_wrapper``.
+These arguments are the ``context`` and ``event`` provided to the AWS Lambda
+handler. With these arguments, Datadog can obtain information to provide
+monitoring for metrics, traces, and logs.
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+This integration is configured automatically. The `datadog_lambda` package
+calls ``patch_all`` when ``DD_TRACE_ENABLED`` is set to ``true``.
+It is not recommended to instantiate this integration.
+
+
+Configuration
+~~~~~~~~~~~~~
+
+.. important::
+
+     No configuration is available by using ``ddtrace.config``.
+     Nonetheless, you can configure some features with environment variables.
+
+.. py:data:: DD_APM_FLUSH_DEADLINE
```

Comment:
What unit is the value? Can we document it here. Ideally, we could include the unit in the variable name. eg: 
```suggestion
.. py:data:: DD_APM_FLUSH_DEADLINE_MILLISECONDS
```

(https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_TRACE_AGENT_TIMEOUT_SECONDS)